rfc/rfc017-distributed-network-grpc-v2.md

```diff
@@ -57,6 +65,21 @@ As specified by RFC015, the node MUST authenticate the peer's node DID as follow
 1. Resolve peer's node DID to its corresponding DID document.
 2. Assert that one of the `NutsComm` endpoint's host matches (one of) the `dNSName` SANs in the peer's TLS client certificate.
 
+To prevent bootstrapping issues, a node MAY have an authenticated and an anonymous connection (both sides are unauthenticated) with a peer at the same time. 
+To create an anonymous connection, a peer MUST NOT send its `nodeDID` and MUST ignore any `nodeDID` returned by the peer.  
```

Comment:
Why the latter? If I never create a DID for my node (which is the case for my dev node), I can still have a connection on which I authenticated the remote node. We need clarification on connection auth status, which kinds do we have and what does it imply (e.g. both sides unauthenticated, one side authenticated, etc).

Reply:
Before we accepted the bootstrap connection as authenticated even if authentication failed. This is undesirable for obvious reason, but solved an issue where the node does not have the data it needs to perform authentication (yet) and can't get the data because authentication fails. 

Making the bootstrap connection anonymous and allowing a second connection through service discovery solves the bootstrap issue without granting the bootstrap connection any extra privileges.

Reply:
will rewrite to make this clearer